clickhouse/ebs.tf

```
@@ -19,7 +19,7 @@ resource "aws_ebs_volume" "keeper" {
   for_each          = local.keeper_nodes
   availability_zone = module.vpc.private_subnet_objects[each.value.subnet_index].availability_zone
   size              = var.keeper_volume_size
-  type              = var.keeper_volume_type
+  type              = "gp3"
   throughput        = 125
   iops              = 3000
```

I think these need to follow the pattern:

```suggestion
  iops              = var.keeper_volume_type == "gp3" ? 3000 : null
```